config/hooks/pre-commit

```diff
@@ -19,6 +19,8 @@ if 'VIRTUAL_ENV' in os.environ:
     sys.path.insert(0, site_packages)
 
 
+PRETTIER_VERSION = "1.2.2"
```

Comment:
That'd probably be more reliable, no? Assuming that you 100% of the time pin explicitly and don't do like, `^2` or other non-explicit identifiers.